gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,122 @@
+        family.
+      </p>
+      <h3 className="nx-h3 nx-tile__subsection-header">Props</h3>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Prop</th>
+            <th className="nx-cell nx-cell--header">Type</th>
+            <th className="nx-cell nx-cell--header">Required</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">productInfo</td>
+            <td className="nx-cell">object</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              An object containing at least one field: <code className="nx-code">name</code>, the product
+              name to display in the header. Additionally, this object may contain
+              a <code className="nx-code">version</code> field holding the product's version number to display.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">homeLink</td>
+            <td className="nx-cell">string</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              A URL (typically relative) that navigates to the home page of the application. If this prop is
+              present, the hexagon logo in the header will be a link to this home page.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">links</td>
+            <td className="nx-cell">array</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              An optional array of objects describing navigation links to display in the middle section of the header.
+              Each link object should contain a <code className="nx-code">name</code> to be displayed and
+              a <code className="nx-code">href</code> for the link. Additionally, at most one link should include
+              a <code className="nx-code">current</code> flag set to true indicating that it should be styled as
+              the currently active link.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell">children</td>
+            <td className="nx-cell">ReactNode</td>
+            <td className="nx-cell">No</td>
+            <td className="nx-cell">
+              Optional additional JSX content that will be displayed at the right end of the header. Note that there
+              are two sections of extra content divided with a vertical line. Which section(s) are used is a design
+              consideration.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h3>Modal Style Classes</h3>
+      <table className="nx-table nx-table--gallery-props">
```

Comment:
This table is broken: 
![image](https://user-images.githubusercontent.com/3630091/97920793-f964e080-1d27-11eb-8dea-7ccb18b03cf9.png)

Reply:
> This table is broken:
> ![image](https://user-images.githubusercontent.com/3630091/97920793-f964e080-1d27-11eb-8dea-7ccb18b03cf9.png)

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/5a9a466d0bfa8eab7298f9ffb38cbb410c53ca6e